conda-forge.yml

```diff
@@ -4,6 +4,9 @@ build_platform:
   linux_aarch64: linux_64
   linux_ppc64le: linux_64
   osx_arm64: osx_64
+provider:
+  linux_aarch64: azure
+  linux_ppc64le: azure
```

Comment:
These can be removed again if we keep cross-compiling.

Reply:
actualité, we need to cross compile since we were using pre built bazel to break a circular dependency